framework/aws-kms/aws-kms-hierarchical-keyring.md

```diff
@@ -293,18 +293,18 @@ When the hierarchical keyring receives an OnEncrypt request,
 the cache entry identifier MUST be calculated as the first 32 bytes of the
 SHA-512 hash of the following byte strings, in the order listed:
 
-| Field                    | Length (bytes) | Interpreted as |
-| ------------------------ | -------------- | -------------- |
-| Length of branch-key-id  | 3              | UInt8          |
-| branch-key-id            | Variable       | UTF-8 Encoded  |
-| Null Byte                | 1              | `0x00`         |
-| Constant string "ACTIVE" | 6              | UTF-8 Encoded  |
+| Field                    | Length (bytes) | Interpreted as         |
+| ------------------------ | -------------- | ---------------------- |
+| Length of branch-key-id  | 3              | UInt8                  |
+| SHA-512(branch-key-id)   | 64             | SHA-512(UTF-8 Encoded) |
```

Comment:
```suggestion
| SHA-512(branch-key-id)   | 64             | bytes |
```

Reply:
Yeah I was wondering how we should write this. This is helpful. Thanks!